README.md

```diff
@@ -1,4 +1,19 @@
-# Ashley, a self-hosted alternative discussion forum for OpenEdx
+# Ashley, a self-hosted alternative discussion forum for learning
+
+## Overview
+
+`Ashley` is a forum designed to be integrated into `Learning Management Systems` (LMS).
+
+It is a multi-tenant service intended to be operated independently. It can be integrated to any LMS (like [Open edX](https://open.edx.org) or [Moodle](https://moodle.org)), thanks to the [LTI](https://en.wikipedia.org/wiki/Learning_Tools_Interoperability) standard for interoperability.
+
+`Ashley` is built on top of [Django-machina](https://github.com/ellmetha/django-machina/), a powerful forum engine for `Django`.
+
+## Quick preview
+
+Here is an example integration of Ashley in `Open edX` :
+
+![demo](docs/demo.gif)
```

Comment:
I'm disappointed there's no :tada: emoji in your demo :crying_cat_face:

Reply:
You're right ... :cry: 
I'll record another demo for the next release :wink: